pkgs/development/python-modules/pyevmasm/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildPythonPackage
+, fetchPypi
+, future
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "pyevmasm";
+  version = "0.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1xpwffnjv65rjr3w7i6v29n2b6a9h5xmsjp1qpxc9ysvsbi38cqv";
+  };
+
+  propagatedBuildInputs = [ future ];
```

Comment:
Does this have built-in tests? ``setuptools test`` doesn't fail if tests are missing, AFAIK.